### PR TITLE
Provide a way to override interop members

### PR DESCRIPTION
**Pull request by sebastienros**
Open to discussion. It might be better to refactor the internal type and dictionary to expose them directly.

### CODE REVIEW

**Comment by sebastienros** [comment]
Thiking also that an overload that return a JsValue instead of a property descriptor, or an object, might be better suited since it's called in CLR land, for CLR instances.

File: Jint/Engine.cs [hunk 1]
```diff
@@ -239,6 +239,12 @@ public Engine(Action<Engine, Options> options)
             ClrTypeConverter = new DefaultTypeConverter(this);
         }
 
+        public void SetMemberAccessor(Type type, string name, Func<Engine, object, PropertyDescriptor> accessor)
```
Comment (lahma) [inline Jint/Engine.cs:242]: Is there a need to change behavior runtime or would it make sense to do these via options when initializing engine?

**Comment by lahma** [comment]
If you look at how object wrapping is configured:

https://github.com/sebastienros/jint/blob/dev/Jint/Options.cs#L78-L87

And then used:

https://github.com/sebastienros/jint/blob/dev/Jint/Native/JsValue.cs#L349-L351

I think allowing to call such callback and if it returns non-null the result should be used instead the one created in normal logic? It would make the API similar.

**Comment by sebastienros** [comment]
Will try to implement a callback instead.

**Comment by sebastienros** [comment]
Updated

**Comment by lahma** [comment]
Looks good!

**Pull request merged**